tests/src/Interop/ICustomMarshaler/MultipleALCs/CustomLoadContext.cs

```diff
@@ -0,0 +1,48 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.Loader;
+
+class CustomLoadContext : AssemblyLoadContext
+{
+    private string unmanagedDirectory;
+
+    internal CustomLoadContext(string unmanagedDirectory)
+    {
+        this.unmanagedDirectory = unmanagedDirectory;
+    }
+
+    protected override Assembly Load(AssemblyName assemblyName)
+    {
+        string assemblyPath = Path.Combine(".", assemblyName.Name) + ".dll";
+        if (File.Exists(assemblyPath))
+        {
+            return LoadFromAssemblyPath(assemblyPath);
```

Comment:
Using a relative path seems to be making an assumption the test is run from the CWD.  It probably is the case during CI, but I have definitely run tests manually w/o meeting this assumption.

`LoadFromAssemblyPath` is supposed to require a fully qualified path, so this should throw if it is called.